server/sample/worlds/sample.json

```diff
@@ -24,10 +24,13 @@
     ]
   },
   "permissions": {
+    "world.api": ["admin", "api"],
+    "world.secrets": ["admin", "api"],
     "world.update": ["admin"],
     "world.announce": ["admin"],
     "room.create": ["admin"],
     "room.announce": ["admin"],
+    "room.inspect": ["admin"],
```

Comment:
What does ``room.inspect`` do?

Reply:
I'll double-check, probably nothing after I added world.secrets. The idea was something like "can see room configuration" as opposed to a possible "room.view" (attendee can see that room exists)